stubs/jmespath/jmespath/visitor.pyi

```diff
@@ -1,49 +1,57 @@
 from typing import Any
+from typing_extensions import TypedDict
+
+from jmespath.functions import Functions
 
 class Options:
     dict_cls: Any
```

Comment:
Cf. this comment in the source code here: https://github.com/jmespath/jmespath.py/blob/bbe7300c60056f52413603cf3e2bcd0b6afeda3d/jmespath/visitor.py#L62-L69

It does have to be a _mutable_ mapping due to the `TreeInterpreter.visit_multi_select_dict()` method (`TreeInterpreter._dict_cls` is set to whatever `options.dict_cls` is in `TreeInterpreter.__init__`): https://github.com/jmespath/jmespath.py/blob/bbe7300c60056f52413603cf3e2bcd0b6afeda3d/jmespath/visitor.py#L233-L236

(You'll need to import `MutableMapping` from `collections.abc`)

```suggestion
    dict_cls: Callable[[], MutableMapping[Any, Any]]
```